### ISSUE TITLE
Using a console will disable all Dear Imgui Buttons in windows 7.

### ISSUE THREAD

**Issue by whiskasdietgirl** [ocornut/imgui#4710]
Im using version v1.76.
Im making an app using 1 pc and 1 laptop using the same exact solution in both.
Its working perfectly on both, shows the menus, buttons work, everything is ok.

However, the problem starts when i try to enable a console in the laptop with windows 7.
It will disable the clicks on the buttons (although it still renders the buttons).
I have no idea why this is happening but it was always like that.
In the windows 10 pc this doesnt happen. The console works, and the imgui buttons are still clickable.
Is there any solution for this? What should i do?
It would be nice to be able to enable the console on this windows 7 laptop so it would be easier to debug certain issues.

So basically i must comment out this 2 lines on laptop 2 with windows 7, if i want my buttons to work in there:
```
		//AllocConsole();
		//freopen("CONOUT$", "w", stdout);
```
Pls help.

**Comment by ocornut** [comment]
Didn't follow requested guidelines or issue template.  Please make sure you have read the guidelines very carefully.
Closing.
